frontend/src/bundles/candidate/components/candidate-modal/candidate-modal.tsx

```diff
@@ -0,0 +1,297 @@
+import {
+    Add as AddIcon,
+    DeleteOutline as DeleteIcon,
+} from '@mui/icons-material';
+import { FormHelperText } from '@mui/material';
```

Comment:
We have component. Import from '~/bundles/common/components/components.js';

Reply:
fixed